### ISSUE TITLE
Fix 4.1% of unknown files; Classify 5,156 medical imaging files ( .dcm)

### ISSUE THREAD

**Issue by raprasad** [IQSS/dataverse#3335]
- similar to #3333

---

5,156 of files with _known_ extension `.dcm` are marked as _unknown_.  
- "DICOM (an acronym for Digital Imaging and Communications in Medicine) is the most common standard for receiving medical scans"

Fixes:
- [x] ~~**Now**: Mark these appropriately, e.g. medical images.~~
- [ ] **Later**: Re-crawl, pulling metadata and producing thumbnails
  - http://pydicom.readthedocs.io/en/stable/index.html
  - Depends on #3088

---

``` json
{
        "extension": ".dcm",
        "count": 5156,
        "total_count": 124357,
        "percent_string": "4.146%"
 },
```

**Comment by mheppler** [comment]
Related to #2202

Also make sure these files have the proper MIME type to display in the cards and facets.

**Comment by pdurbin** [comment]
Closing in favor of #2202
